### DISCUSSION TITLE
Firebase Auth possible?

### DISCUSSION THREAD

**Discussion by webwurm** [Budibase/budibase#6420] [General]
I like the Budibase-tool very much and I'd like to use it for my project. But I have one criteria, that is important to me. Maybe somebody can tell me, if this is somehow possible:

I have an app that uses Firebase as authentification-plattform for e.g. email-auth, Google and Apple. So all my users are stored there. I want to give my users a "public backend" (I don't know a better name for it...) on the web, where they can login and do stuff.

Is this somehow possible with Budibase? That it takes the auth-data from Firebase and use it in my Budibase-app as logins?

Thanks in advance!

**Comment by melohagan** [comment]
Hi @webwurm 

Thank you for the kind words!

I don't think this scenario is possible, however Budibase does offer Google's SSO.
